compiler/test/dotty/tools/dotc/transform/PatmatExhaustivityTest.scala

```diff
@@ -18,7 +18,7 @@ import org.junit.Test
 class PatmatExhaustivityTest {
   val testsDir = "tests/patmat"
   // stop-after: patmatexhaust-huge.scala crash compiler
-  val options = List("-color:never", "-Ystop-after:crossCast", "-Ycheck-all-patmat", "-classpath", TestConfiguration.basicClasspath)
+  val options = List("-color:never", "-Ystop-after:explicitSelf", "-Ycheck-all-patmat", "-classpath", TestConfiguration.basicClasspath)
```

Comment:
That explains the mystery of the added lines, I suppose? Good catch!

Reply:
Yes, most failures come from this change. The test case `i8922c.scala` and `i4030.scala` are due to type inference change.